src/source/source_cache.js

```diff
@@ -179,15 +178,6 @@ class SourceCache extends Evented {
         for (const id in this._tiles) {
             if (this._isIdRenderable(+id, symbolLayer)) renderables.push(this._tiles[id]);
         }
-        if (symbolLayer) {
```

Comment:
`symbolLayer` argument is no longer used so we should remove it from signature / calls.